### PR TITLE
REMO-39: Create workflow for test and build

### PR DESCRIPTION
**Pull request by chalalala**

### CODE REVIEW

File: .github/workflows/run-build-and-test.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      with:
+        version: 8.6.10
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'pnpm'
+    - run: pnpm install
+    - run: pnpm build
+    - run: pnpm test
```
Comment (thuutri2710) [inline .github/workflows/run-build-and-test.yml:37]: i dont think we need `pnpm test` here

Comment (chalalala) [reply .github/workflows/run-build-and-test.yml:37]: The main purpose of this workflow is to check if all the tests are passed and the build can be run successfully with the changes of the PR, so I think it's necessary to have `pnpm test` here 🤔

Comment (thuutri2710) [reply .github/workflows/run-build-and-test.yml:37]: I dont say that we dont need `pnpm test`. Just want to move it to another job

Comment (chalalala) [reply .github/workflows/run-build-and-test.yml]: Updated in [f79eed4](https://github.com/chalalala/remo/pull/10/commits/f79eed484fc05ff743697b8a3424c060d4df8c92)

**Review by thuutri2710 — APPROVED** [review]
LGTM

**Pull request merged**